ui/app/store/actions.js

```diff
@@ -169,33 +137,22 @@ const actions = {
   cancelTxs,
   completedTx: completedTx,
   txError: txError,
-  nextTx: nextTx,
-  editTx,
-  previousTx: previousTx,
-  cancelAllTx: cancelAllTx,
-  viewPendingTx: viewPendingTx,
-  VIEW_PENDING_TX: 'VIEW_PENDING_TX',
   updateTransactionParams,
   UPDATE_TRANSACTION_PARAMS: 'UPDATE_TRANSACTION_PARAMS',
-  setNextNonce,
   SET_NEXT_NONCE: 'SET_NEXT_NONCE',
   getNextNonce,
   // send screen
   UPDATE_GAS_LIMIT: 'UPDATE_GAS_LIMIT',
   UPDATE_GAS_PRICE: 'UPDATE_GAS_PRICE',
   UPDATE_GAS_TOTAL: 'UPDATE_GAS_TOTAL',
-  UPDATE_SEND_FROM: 'UPDATE_SEND_FROM',
   UPDATE_SEND_HEX_DATA: 'UPDATE_SEND_HEX_DATA',
   UPDATE_SEND_TOKEN_BALANCE: 'UPDATE_SEND_TOKEN_BALANCE',
   UPDATE_SEND_TO: 'UPDATE_SEND_TO',
   UPDATE_SEND_AMOUNT: 'UPDATE_SEND_AMOUNT',
-  UPDATE_SEND_MEMO: 'UPDATE_SEND_MEMO',
   UPDATE_SEND_ERRORS: 'UPDATE_SEND_ERRORS',
   UPDATE_MAX_MODE: 'UPDATE_MAX_MODE',
   UPDATE_SEND: 'UPDATE_SEND',
   CLEAR_SEND: 'CLEAR_SEND',
-  OPEN_FROM_DROPDOWN: 'OPEN_FROM_DROPDOWN',
```

Comment:
This is still referenced in the `send` store, and in the tests for that store.

Reply:
And likewise for `CLOSE_FROM_DROPDOWN`

Reply:
I believe that's a different `OPEN_FROM_DROPDOWN`

Reply:
And a different `CLOSE_FROM_DROPDOWN`

Reply:
Oh! You're right.

Those are also unused, but yes they are distinct.

Reply:
I can remove them in a separate PR 😄

Reply:
Sure haha you can have those

Reply:
https://github.com/MetaMask/metamask-extension/pull/7700